test/boolean_complex_test.cpp

```diff
@@ -1084,9 +1083,8 @@ TEST(BooleanComplex, SimpleOffset) {
     if (len < 0.03) continue;
     manifold::Manifold origin_cyl = manifold::Manifold::Cylinder(len, 1, 1, 8);
     vec3 evec(-1 * edge.x, -1 * edge.y, edge.z);
-    manifold::Manifold rotated_cyl =
-        origin_cyl.Transform(manifold::RotateUp(evec));
-    manifold::Manifold right = rotated_cyl.Translate(ev1);
+    quat q = rotation_quat(normalize(evec), vec3(0, 0, 1));
```

Comment:
@pca006132 I was a little suprised I didn't need `la::rotation_quat` or `manifold::la::rotation_quat`. I recall you made a comment about my namespacing - did I do it wrong?

Reply:
I think this is a weird thing called [Argument-dependent lookup](https://en.cppreference.com/w/cpp/language/adl) for functions, which basically adds the argument type namespaces into the lookup search path. I don't think we have anything wrong with our namespacing.

Reply:
Oh, that's cool! Thanks for the explanation.